plugins/plugin_run/project_run.py

```diff
@@ -166,7 +166,12 @@ def _run_ios_app(self, ios_app_path):
 
         try:
             # run the simulator
-            self._run_cmd('xcrun instruments -w "%s"' % simulator_id)
+            xcode_version = cocos.get_xcode_version()
+            xcode9_and_upper = cocos.version_compare(xcode_version,">=",9)
+            if xcode9_and_upper:
+                self._run_cmd('xcrun instruments -w "%s" -p "Leaks"' % simulator_id)
```

Comment:
the new version of `xcrun` need to finger a `-p` parameter, or it will report a error, tested on Xcode 9.3

Reply:
I think `Leaks` is not correct here, it will slow down the performance.